generated/nidaqmx/base_interpreter.py

```diff
@@ -0,0 +1,1630 @@
+    def get_scale_attribute_int_32(self):
+        pass
+
+    def get_scale_attribute_string(self):
+        pass
+
+    def get_self_cal_last_date_and_time(self):
+        pass
+
+    def get_start_trig_timestamp_val(self):
+        pass
+
+    def get_start_trig_trig_when(self):
+        pass
+
+    def get_sync_pulse_time_when(self):
+        pass
+
+    def get_system_info_attribute_string(self):
+        pass
+
+    def get_system_info_attribute_u_int_32(self):
+        pass
+
+    def get_task_attribute_bool(self):
+        pass
+
+    def get_task_attribute_string(self):
+        pass
+
+    def get_task_attribute_u_int_32(self):
+        pass
+
+    def get_timing_attribute_bool(self):
+        pass
+
+    def get_timing_attribute_double(self):
+        pass
+
+    def get_timing_attribute_ex_bool(self):
+        pass
+
+    def get_timing_attribute_ex_double(self):
+        pass
+
+    def get_timing_attribute_ex_int_32(self):
+        pass
+
+    def get_timing_attribute_ex_string(self):
+        pass
+
+    def get_timing_attribute_ex_timestamp(self):
+        pass
+
+    def get_timing_attribute_ex_u_int_32(self):
+        pass
+
+    def get_timing_attribute_ex_u_int_64(self):
+        pass
+
+    def get_timing_attribute_int_32(self):
+        pass
+
+    def get_timing_attribute_string(self):
+        pass
+
+    def get_timing_attribute_timestamp(self):
+        pass
+
+    def get_timing_attribute_u_int_32(self):
+        pass
+
+    def get_timing_attribute_u_int_64(self):
+        pass
+
+    def get_trig_attribute_bool(self):
+        pass
+
+    def get_trig_attribute_double(self):
+        pass
+
+    def get_trig_attribute_double_array(self):
+        pass
+
+    def get_trig_attribute_int_32(self):
+        pass
+
+    def get_trig_attribute_int_32_array(self):
+        pass
+
+    def get_trig_attribute_string(self):
+        pass
+
+    def get_trig_attribute_timestamp(self):
+        pass
+
+    def get_trig_attribute_u_int_32(self):
+        pass
+
+    def get_watchdog_attribute_bool(self):
+        pass
+
+    def get_watchdog_attribute_double(self):
+        pass
+
+    def get_watchdog_attribute_int_32(self):
+        pass
+
+    def get_watchdog_attribute_string(self):
+        pass
+
+    def get_write_attribute_bool(self):
+        pass
+
+    def get_write_attribute_double(self):
+        pass
+
+    def get_write_attribute_int_32(self):
+        pass
+
+    def get_write_attribute_string(self):
+        pass
+
+    def get_write_attribute_u_int_32(self):
+        pass
+
+    def get_write_attribute_u_int_64(self):
+        pass
+
+    def is_task_done(self):
+        pass
+
+    def load_task(self):
+        pass
+
+    def read_analog_f_64(self):
+        pass
+
+    def read_analog_scalar_f_64(self):
+        pass
+
+    def read_binary_i_16(self):
+        pass
+
+    def read_binary_i_32(self):
+        pass
+
+    def read_binary_u_16(self):
+        pass
+
+    def read_binary_u_32(self):
+        pass
```

Comment:
Snake case conversion of `Int32`, `UInt32`, `F64`, etc. resulted in extra underscores.

Reply:
Sure. I have removed leading underscores in function names that has datatypes.